ilc/client/isActiveFactory.spec.js

```diff
@@ -137,15 +124,13 @@ describe('is active factory', () => {
         chai.expect(triggerAppChange.calledOnce).to.be.true;
         chai.expect(handlePageTransaction.secondCall.calledWithExactly(slotName, slotWillBe.default)).to.be.true;
 
-        removeIlcConfig();
-
         chai.expect(isActive()).to.be.true;
 
         dispatchSingleSpaAppChangeEvent();
 
         await clock.runAllAsync();
 
         chai.expect(triggerAppChange.calledOnce).to.be.true;
-        chai.expect(handlePageTransaction.calledTwice).to.be.true;
+        chai.expect(handlePageTransaction.calledThrice).to.be.true;
```

Comment:
If it should call thrice, I guess, we have to expect `calledWithExactly` some arguments as we have above for second call and first call, because now I don't understand why it should call thrice

Reply:
Switched to `calledWithExactly `